### DISCUSSION TITLE
Allowing mDNS through Wireguard VPN?

### DISCUSSION THREAD

**Discussion by cerahmed** [wg-easy/wg-easy#443] [General]
Hi there,

I've followed the instructions in this [reddit ](https://www.reddit.com/r/WireGuard/comments/g80bxf/comment/hr9bqi4/?utm_source=share&utm_medium=web2x&context=3)using my other original WG (not wg-easy) installation, and with the help of [mdns-repeater](https://github.com/geekman/mdns-repeater), it worked!
Now I'm trying to do the same for wg-easy. However, due to the fact that wg0 interface is hidden behind docker0, I could not get it to work.

After a bit of looking around, I found that I can use "--net host" to make docker container share the name network namespace as the host (RPi in my case). So I eliminated all port forwarding rules in the "docker run" command, as they are ignored anyways.

Now wg0 is accessible as expected by the host, however, for some the VPN is not working. I made sure that both tcp and udp ports are exposed, and in fact I can reach 51821 webpage.

I suspect that it has something to do with POST UP argument, which I tried to change with no luck.

Any idea?

**Comment by pheiduck** [comment]
Please try the latest image: `ghcr.io/wg-easy/wg-easy:latest`
